### ISSUE TITLE
No Folder Creation and Asks to save each image

### ISSUE THREAD

**Issue by cewong2** [mnao305/fantia-dl-tool#84]
On 3.0 from the Chrome Extension Store

When you click the download button, it doesn't create a folder structure, and it pops up to ask to save each individual image

**Comment by mnao305** [comment]
Thanks for reporting it.
Hmm, I couldn't reproduce it in my environment. I'll look into it some more.

Does it occur in all posts?
Also, if you don't mind me asking, could you tell me what environment you are experiencing the problem in? (OS, browser version).

**Comment by asdf1324** [comment]
Cannot reproduce the issue on Edge 93.0.910.5 (Official build) dev (64-bit) (Chrome/93.0.4521.0), Windows  - the images seem to be saved in the correct place and it definitely doesn't show prompts for every individual images. The version of the extension is 3.2.0 (which appears to be the latest available from the store.)

Capture of the edge://version page (The equivalent in Chrome would be chrome://version).
![image](https://user-images.githubusercontent.com/19871820/121772328-b6b59180-cbaf-11eb-95db-bf24e652fe95.png)

**Comment by cewong2** [comment]
Sorry for the slow reply, I was using it on MacOS, Mojave using Brave Browser

**Comment by mnao305** [comment]
@cewong2 
Thanks for letting me know your usage environment!
I tried it with Brave Browser and got a confirmation dialog.

I checked it out, and it seems that the browser settings are causing the bad behavior.
Please go to brave://settings/downloads and turn off "Ask where to save each file before downloading" and try again.

![image](https://user-images.githubusercontent.com/32170530/121797051-a1e10880-cc58-11eb-8c41-bcd3b3dc266c.png)

**Comment by cewong2** [comment]
I forgot about that for pop up saves. It seems on Brave, it won't do the folder creation and directory structure, but it will save all the images.

**Comment by mnao305** [comment]
Oops, did you fail to create the folder?
In my environment, the folders are created in the correct hierarchy. (macOS Big Sur, Brave Browser)

**Comment by cewong2** [comment]
yeah it didn't create the folder, only downloaded all the images, (Mojave, Brave). It works with (Mojave, Chrome)

**Comment by mnao305** [comment]
Hmm, I just tried it on macOS Mojave in a virtual environment and it seems to be working fine.
I have no idea what could be causing that problem...
